core/php8.2Action/Dockerfile

```diff
@@ -0,0 +1,93 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# build go proxy from source
+ARG GO_PROXY_BASE_IMAGE=golang:1.18
+FROM $GO_PROXY_BASE_IMAGE AS builder_source
+ARG GO_PROXY_GITHUB_USER=nimbella-corp
+ARG GO_PROXY_GITHUB_BRANCH=dev
+RUN git clone --branch ${GO_PROXY_GITHUB_BRANCH} \
+  https://github.com/${GO_PROXY_GITHUB_USER}/openwhisk-runtime-go /src ;\
+  cd /src ; env GO111MODULE=on CGO_ENABLED=0 go build main/proxy.go && \
+  mv proxy /bin/proxy
+
+# or build it from a release
+FROM $GO_PROXY_BASE_IMAGE AS builder_release
+ARG GO_PROXY_RELEASE_VERSION=1.15@1.17.0
```

Comment:
similar comment as above-- may be out of scope for this PR, but I do see what appear to be newer go runtimes with action loop in the OW repo.

Reply:
We never actually use the prereleased version. This is legacy, lemme kill it.